ReadMe Visualization: Links with Pictures (#5)
asam-ev/.github: profile/README.md

## ASAM Projects

| ASAM QC FRAMEWORK            | ASAM OSI              | ASAM OpenMATERIAL         | ASAM ODS |
|:----------------------------:|:---------------------:|:------------------------:|:------------------------:|

> | [<img src="/doc/img/ASAM Quality Checker Framework.png" width="100" />](https://github.com/asam-ev/qc-framework) | [<img src="/doc/img/OSI.jpg" width="100" />](https://github.com/OpenSimulationInterface/open-simulation-interface) | [<img src="/doc/img/sharepoint_logo_openMaterial.png" width="100" />](https://github.com/asam-ev/OpenMATERIAL) | [<img src="/doc/img/ODS.png" width="100" />](https://github.com/asam-ev/ASAM-ODS-Interfaces) |

Reviewer: Would it be feasible to link to logos in a central place so that we don't duplicate them all over the place? Maybe in a separate repo.? Or directly somewhere else where we maintain marketing assets?

Author: Unfortunately I´ve been trying to use the URL directly from our SharePoint:

https://asamev.sharepoint.com/Freigegebene%20Dokumente?viewpath=%2FFreigegebene%20Dokumente&id=%2FFreigegebene%20Dokumente%2F11%2E%20Technology%20Management%2Finternal%2FGitHub%2FGitHub%5FProjeckt%5FLogos&viewid=39ab6b2a%2D6b8d%2D40ca%2D970d%2D71ee86952224

But the the picture is not recognized in Markdown. I can provide a link to click on, but not visualize the image.
I´ve tried with different methods and all were unsuccessful. Maybe it is a good idea just to maintain the pictures uses directly on the repo, under the "img" folder and reference from there.

Reviewer: No worries, then let's go with your approach.

More information about active projects can be found on our webpage [www.asam.net](https://www.asam.net/active-projects/projects/).

> And also quick-links to other resources and tools:

Reviewer: Sentence starting with "And".
```suggestion
Quick-links to other resources and tools:
```

A quality checker framework that allows the configuration, execution and reporting of checks. Checks are based on rules defined by ASAM standards that allow users to validate conformity of files and implementations against ASAM standards.

> - [qc-framework](https://github.com/asam-ev/qc-framework) - The main repository for the ASAM Quality Checker formed by a collection of C++ and Python components responsible for running the checker bundles.

Reviewer: does the base framework also have python components or is this actually the baselib-py?

Author: Yes, 90% is C++ but there are some modules that are python

- [qc-baselib-py](https://github.com/asam-ev/qc-baselib-py) - The Python Base library repository implements a Python interface for interacting with the configuration files and the results files from the ASAM Quality Checker framework.

> - [qc-opendrive](https://github.com/asam-ev/qc-opendrive) - Specific ASAM OpenDRIVE repository which implements his official ASAM OpenDRIVE checker bundle.
> - [qc-openscenarioxml](https://github.com/asam-ev/qc-openscenarioxml) - Specific ASAM OpenSCENARIO XML repository which implements his official ASAM OpenSCENARIO XML checker bundle.
> - [qc-otx](https://github.com/asam-ev/qc-otx) - Specific Open Test sequence eXchange (OTX) repository which contains a short representative list of check examples (it shall not be a reference implementation).

Reviewer: ```suggestion
- [qc-opendrive](https://github.com/asam-ev/qc-opendrive) - The official ASAM checker bundle for ASAM OpenDRIVE.
- [qc-openscenarioxml](https://github.com/asam-ev/qc-openscenarioxml) - The official ASAM checker bundle for ASAM OpenSCENARIO XML.
- [qc-otx](https://github.com/asam-ev/qc-otx) - A checker bundle implementing a sample of rules for the standard ASAM OTX Extensions (Open Test sequence eXchange). Note that this is not to be considered an exhaustive checker bundle.
```

Reviewer: I will leave this for you to accept this suggestion :)

### ASAM OSI® (Open Simulation Interface)
ASAM OSI® (Open Simulation Interface) is a specification for interfaces between models and components of a distributed simulation. OSI is strongly focused on the environmental perception of automated driving functions.

> - [Documentation](https://opensimulationinterface.github.io/osi-antora-generator/asamosi/latest/specification/index.html)

Reviewer: ```suggestion
- [Documentation](https://opensimulationinterface.github.io/osi-antora-generator/asamosi/latest/specification/index.html) - The ASAM OSI specification
```
